modules/nf-core/rgi/main/tests/main.nf.test

```diff
@@ -16,11 +16,11 @@ nextflow_process {
             script "modules/nf-core/untar/main.nf"
             process {
             """
-            file('https://card.mcmaster.ca/latest/data', checkIfExists: true).copyTo('data.tar.gz')
+            file('https://card.mcmaster.ca/latest/data', checkIfExists: true).copyTo('card-data.tar.bz2')
```

Comment:
Insert raised eyebrow emoji here

Reply:
yeah, I don't like this test, but no time to deal with it nicely, I just put the right extension for the actual file